Dockerfile

```diff
@@ -2,36 +2,33 @@ FROM node:10.13-alpine
 
 WORKDIR /app
 
+ENV PORT 3005
+EXPOSE 3005
+
 # Ensure COMMIT_HASH is present
 ARG COMMIT_HASH
 RUN test -n "$COMMIT_HASH"
+RUN echo $COMMIT_HASH > COMMIT_HASH.txt
 
 # Install system dependencies
-RUN apk add \
+RUN apk add --no-cache --quiet \
   bash \
   curl \
   dumb-init \
   git
 
-# Add deploy user
-RUN adduser -D -g '' deploy
-
 # Install the packages
 COPY package.json yarn.lock ./
 RUN yarn install
 
+# Add deploy user
+RUN adduser -D -g '' deploy
```

Comment:
I think this can move up as well as there's nothing changing here build to build. Perhaps joined with the `apk` RUN instruction?

Reply:
I'd rather leave things on separate lines so that it is clearer to those reading, is there any advantage to putting them into one command besides being shorter?

Reply:
Each separate docker instruction results in a separate docker layer, which adds complexity and time to docker pushes and pulls.

One option would be to retain the separate comment lines but still collapse to a single RUN instruction:

```dockerfile
# Install system dependencies
RUN apk add --no-cache --quiet \
      bash \
      curl \
      dumb-init \
      git && \
  # Add deploy user \
  adduser -D -g '' deploy
```